gradle-baseline-java-config/resources/findbugs/excludeFilter.xml

```diff
@@ -18,4 +18,16 @@
             <Bug pattern="URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD"/> <!-- @Rule ivars can be public and unused -->
         </Or>
     </Match>
+
+    <!-- Ignore the following bug patterns in Immutables generated code -->
+    <!-- (i.e., classes beginning with 'Immutable' and inner classes of same) -->
+    <Match>
+        <Class name="~.*Immutable.*(\$.*)?" />
```

Comment:
yes, I was also worried about that (non-generated classes starting with Immutable).

actually it looks like Immutables has added FB suppressions in 2.1.12 (released 2 days ago!) so I'll try upgrading to that instead.
https://github.com/immutables/immutables/issues/263